### ISSUE TITLE
[BITV] 9.1.3.1h/4.1 - Modal dialog "Create new album" - Both input fields are only labeled with placeholder elements. The value may not be read depending on the browser and assistive technology. The labeling should be done through a linked (possibly accessible hidden) "label" element or through "aria-label". If the "title" attribute is used, the value should match that of "aria-label" or "label" to minimize redundant outputs in screen readers. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/photos#1666]
- [ ] Fix
- [ ] Backport

<img width="628" alt="3ce7b2fe448c1590ea03118ac5bdf862" src="https://user-images.githubusercontent.com/1315170/221935485-3a96607a-17bc-4565-b6c8-bf9843ef2e2f.png">

![0e4660212b80e42b753b1adcc08a8370](https://user-images.githubusercontent.com/1315170/221935517-c3314450-9bef-4775-91eb-f299787f3e06.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-fa5c8d46f9-v4-n1

</details>
